go/teams/hidden/hidden.go

```diff
@@ -338,83 +335,18 @@ func ProcessHiddenResponseFunc(m libkb.MetaContext, teamID keybase1.TeamID, apiR
 }
```

Comment:
There is no committedHiddenLink any more. Maybe we want to update function names?

Reply:
I think there's a lot of function names that imply stuff about a merkle tree, it might be tricky to try to get them all